src/main/java/com/hazelcast/kubernetes/HazelcastKubernetesDiscoveryStrategy.java

```diff
@@ -103,6 +98,40 @@ private String discoverZone() {
         return "unknown";
     }
 
+    /**
+     * Discovers the name of the node which the current Hazelcast member pod is running on.
+     * <p>
+     * Note: NODE_AWARE is available only for the Kubernetes API Mode.
+     */
+    private String discoverNodeName() {
+        if (DiscoveryMode.KUBERNETES_API.equals(config.getMode())) {
+            try {
+                String podName = podName();
+                String nodeName = client.nodeName(podName);
```

Comment:
```suggestion
                String nodeName = client.nodeName(podName());
```
Nit: Consider inlining.